messaginginapp/src/main/java/io/customer/messaginginapp/gist/presentation/GistModalManager.kt

```diff
@@ -14,23 +14,25 @@ internal class GistModalManager : GistListener {
     }
 
     internal fun showModalMessage(message: Message, position: MessagePosition? = null): Boolean {
```

Comment:
can we not just make the method `synchronized`?

Reply:
Updated and retested. Thanks for the suggestion!